repeating_dates.module

```diff
@@ -98,7 +98,19 @@ function repeating_dates_field_widget_settings_form($field, $instance) {
         'date' => t('Date'),
       ),
       '#default_value' => isset($settings['default_stop_mode']) ? $settings['default_stop_mode'] : 'count',
-      );
+    );
+    $form['default_count'] = array(
```

Comment:
@herbdool this is an additional setting, that also has to get a default value in repeating_dates_field_widget_info().

If set there, probably most isset/empty/whatever checks can go.

But we need an update hook for the field instance config (json) files. For all that use the repeating_date_widget.

Reply:
Thanks for the reminder about `hook_field_widget_info()`. I'll update.